pages/common/aws-ecr.md

```diff
@@ -0,0 +1,36 @@
+# aws ecr
+
+> Push, pull, and manage container images.
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/ecr/index.html>.
+
+- Authenticate local container manager to your default registry (username is AWS):
+
+`aws ecr get-login-password --region {{region}} | {{docker login}} --username AWS --password-stdin {{aws_account_id}}.dkr.ecr.{{region}}.amazonaws.com`
```

Comment:
I think we should just stick with Docker here, and not try to cover all others.  Podman for example may not support `--password-stdin`.
```suggestion
- Authenticate local Docker to your default registry (username must be AWS):

`aws ecr get-login-password --region {{region}} | docker login --username AWS --password-stdin {{aws_account_id}}.dkr.ecr.{{region}}.amazonaws.com`
```
(also, doesn't `dkr` in the URL stand for docker? :)

Reply:
should that be ` - Authenticate a local`

Reply:
Having trouble finding anything on .dkr. in the documentation. Assuming everyone uses docker would simplify things.

I might eventually test out ecr with docker alternatives, but right now it wouldn't be a good use of time.

Good idea.

Reply:
@CleanMachine1 what about "Authenticate Docker with your default registry...."

Reply:
Yeah @258204 I like that better. here
```suggestion
- Authenticate Docker with your default registry (username is AWS):

`aws ecr get-login-password --region {{region}} | {{docker login}} --username AWS --password-stdin {{aws_account_id}}.dkr.ecr.{{region}}.amazonaws.com`
```